diff -up u-boot-2026.01/drivers/usb/host/dwc2.c.2~ u-boot-2026.01/drivers/usb/host/dwc2.c
--- u-boot-2026.01/drivers/usb/host/dwc2.c.2~	2026-01-05 21:49:22.000000000 +0100
+++ u-boot-2026.01/drivers/usb/host/dwc2.c	2026-01-29 17:37:29.967686080 +0100
@@ -361,6 +361,8 @@ static void dwc_otg_core_init(struct ude
 
 	writel(usbcfg, &regs->global_regs.gusbcfg);
 
+	mdelay(10);
+
 	/* Program the GAHBCFG Register. */
 	switch (FIELD_GET(GHWCFG2_ARCHITECTURE_MASK, readl(&regs->global_regs.ghwcfg2))) {
 	case GHWCFG2_SLAVE_ONLY_ARCH:
