diff -up u-boot-2021.07/tools/imagetool.c.omv~ u-boot-2021.07/tools/imagetool.c
--- u-boot-2021.07/tools/imagetool.c.omv~	2021-09-13 11:45:58.387624093 +0200
+++ u-boot-2021.07/tools/imagetool.c	2021-09-13 11:46:09.170765142 +0200
@@ -9,6 +9,8 @@
 
 #include <image.h>
 
+struct image_type_params *__start_image_type[], *__stop_image_type[];
+
 struct image_type_params *imagetool_get_type(int type)
 {
 	struct image_type_params **curr;
diff -up u-boot-2021.07/tools/imagetool.h.omv~ u-boot-2021.07/tools/imagetool.h
